BraveShared/BraveStrings.swift

```diff
@@ -17,6 +17,13 @@ import Shared
 
 // swiftlint:disable line_length
 
+extension Strings {
+    /// "BAT" or "BAT Points" depending on the region
+    public static var BAT: String {
+      return Preferences.Rewards.isUsingBAP.value == true ? "BAP" : "BAT"
```

Comment:
So I duplicated some strings to leave BR-UI strings locked as internal.

Not perfect, we might want to rework that at some point

Reply:
Yeah this is OK, but in the end we can probably merge them into 1 struct. Reason it didnt start that way was because BraveRewardsUI used to be in its own repo.